src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -703,9 +703,24 @@ protected virtual ClaimsIdentity CreateClaimsIdentity(JsonWebToken jwtToken, Tok
             return CreateClaimsIdentity(jwtToken, validationParameters, actualIssuer);
         }
 
-        private ClaimsIdentity CreateClaimsIdentity(JsonWebToken jwtToken, TokenValidationParameters validationParameters, string actualIssuer)
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="jwtToken"></param>
+        /// <param name="validationParameters"></param>
+        /// <param name="issuer"></param>
+        /// <returns></returns>
+        protected virtual ClaimsIdentity CreateClaimsIdentity(JsonWebToken jwtToken, TokenValidationParameters validationParameters, string issuer)
```

Comment:
>string issuer) [](http://example.com/codeflow?start=133&length=14)

nit: if issuer can be null, then you could add it to the original method at the end. We were discussing last week if this is a breaking change or not. We do this a lot in ID web to keep API surface smaller and helps with testing.

Reply:
Leaving this pending, on the question below around whether or not issuer could be null.

Reply:
I've refactored a bit the calls and it should be clearer now that it is calling the private method. I 'created' a second virtual instead of changing the original private method that was there with the logic, with both virtual methods calling it.